README.md

```diff
@@ -1805,7 +1805,9 @@ IEEE Access, 2020.
 - <a name="Banerjee20arxiv"></a>Banerjee, S., Wang, Z. W., Chopp, H. H., Cossairt, O., Katsaggelos, A.,  
 *[Quadtree Driven Lossy Event Compression](https://arxiv.org/pdf/2005.00974.pdf)*,  
 arXiv 2020. [PDF](https://arxiv.org/pdf/2005.00974.pdf)
-
+- <a name="Khan21IoT-J"></a>N. Khan, K. Iqbal and M. G. Martini,  
```

Comment:
Thanks. Could you...
- format the names as in a previous entry: last name, initial of name, etc.
- use Khan21jiot as key? (from the letters in the DOI: 10.1109/JIOT.2020.3007866)

Reply:
Thanks for reply, I've changed them.

Reply:
Thanks! If you have a publicly accessible pdf (preprint or similar), it would be good to provide it, too.